package.json

```diff
@@ -8,8 +8,9 @@
   ],
   "scripts": {
     "build": "node build.js",
-    "transform-base": "npx token-transformer data data/transformed-core.json core --expandTypography",
-    "transform": "yarn transform-base"
+    "transform-core": "npx token-transformer data data/transformed-core.json core --expandTypography",
+    "transform-light": "npx token-transformer data data/transformed-light.json core,light core",
```

Comment:
I brought this up in our group chat, but we have some instances of `npm` in our GitHub workflow, do we want to stick with `yarn` for everything and remove `package-lock.json`? We can update this in another MR

Reply:
i think that makes sense! seems like a good clean up